src/config.py

```diff
@@ -216,6 +214,22 @@ class OakDConfig(OakConfig):
     OAK-D component model native config
     """
 
+    VALID_ATTRIBUTES: ClassVar[List[str]] = [
```

Comment:
`VALID_ATTRIBUTES` pattern here seems nice for checking inputs and giving good error msgs if not getting what we expect.